clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v2/description/manifest/KubernetesManifestTraffic.java

```diff
@@ -17,16 +17,14 @@
 package com.netflix.spinnaker.clouddriver.kubernetes.v2.description.manifest;
 
 import lombok.AllArgsConstructor;
-import lombok.Builder;
-import lombok.Data;
-import lombok.NoArgsConstructor;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
 
 import java.util.List;
 
-@Data
-@Builder
 @AllArgsConstructor
-@NoArgsConstructor
+@EqualsAndHashCode
+@Getter
```

Comment:
`@Data` gives you both getters and setters for member fields here. Not sure if there's a difference when it comes to serializing this object.

Reply:
Yeah, removing the setters would make serialization no longer work without annotating the constructor, but we never actually serialize/deserialize this object.

Reply:
:+1: